docs/api/app.md

```diff
@@ -398,6 +398,16 @@ non-minimized.
 This event is guaranteed to be emitted after the `ready` event of `app`
 gets emitted.
 
+### Event: 'desktop-capturer-get-sources'
+
+Returns:
+
+* `event` Event
+* `webContents` [WebContents](web-contents.md)
+
+Emitted when `desktopCapturer.getSources()` is called in the renderer process of `webContents`.
+Calling `event.preventDefault()` will make it throw an error.
```

Comment:
Instead of making this throw an error, can we make it transparent to the caller and return an empty array.

Reply:
That's what I did initially before pushing the PR and then I though an error is more explicit.
What do others think?
cc @nornagon, @deepak1556, @alexeykuzmin